target/linux/octeon/base-files/lib/upgrade/platform.sh

```diff
@@ -68,22 +88,25 @@ platform_do_upgrade() {
 	erlite)
 		kernel=sda1
 		;;
+	itus,shield-router)
+		kernel=ItusrouterImage
+		;;
 	*)
 		return 1
 	esac
 
 	platform_do_flash $tar_file $board $kernel $rootfs
 
 	return 0
-	
 }
 
 platform_check_image() {
 	local board=$(board_name)
 
 	case "$board" in
 	er | \
-	erlite)
+	erlite | \
+	itus,shield-router)
 		local tar_file="$1"
 		local kernel_length=$(tar xf $tar_file sysupgrade-$board/kernel -O | wc -c 2> /dev/null)
```

Comment:
Without boardname and device definition changed to underscore variant, this "sysupgrade-$board" won't work anymore (I suspect).
Consider replacing it by a "sysupgrade-*"

Reply:
Will test